test/he/test_conn.c

```diff
@@ -243,6 +243,44 @@ void test_is_password_set(void) {
   TEST_ASSERT_EQUAL(true, res3);
 }
 
+// Handling Auth Token
+
+void test_set_auth_token(void) {
+  int res1 = he_conn_set_auth_token(&conn, fake_ipv4_packet, sizeof(fake_ipv4_packet));
```

Comment:
nit, there's only a single res here, so we can call this `res` instead of `res1`